src/kraken/std/python/tasks/info_task.py

```diff
@@ -0,0 +1,83 @@
+import re
+import subprocess
+from pathlib import Path
+from typing import List
+
+from kraken.core.api import Project, Property, Task, TaskStatus
+
+from ..buildsystem import PythonBuildSystem
+
+
+def get_python_path() -> str:
+    """Returns Python path"""
+    path = run_command(["which", "python"])
+    if "command not found" in path:  # Windows
+        path = run_command(["where", "python"])
+    return path
+
+
+def get_python_version() -> str:
+    """Returns Python version"""
+    return run_command(["python", "--version"])
```

Comment:
Similarly, we cannot expect `python` to be what we want, so we need to go through the managed environment:

```suggestion
    """Returns the version of the Python interpreter of the Kraken-managed environment."""
    return run_command([self.get_python_path(), "--version"])
```